crates/viewer/re_viewer_context/src/contents.rs

```diff
@@ -65,6 +65,14 @@ impl Contents {
     }
 }
 
+impl TryFrom<&ItemCollection> for Vec<Contents> {
+    type Error = ();
+
+    fn try_from(value: &ItemCollection) -> Result<Self, Self::Error> {
+        value.iter().map(|(item, _)| item.try_into()).collect()
+    }
+}
+
```

Comment:
in the next PR, I'm moving that next to `ItemCollection` instead, because I'll have another similar one but for entities.